scripts/install_prereqs.sh

```diff
@@ -55,6 +55,14 @@ case "$PROJECT-$PLATFORM-$METHOD" in
     bundle exec pod install --project-directory=GoogleUtilities/Example
     ;;
 
+  Functions-*)
+    bundle exec pod repo update
+    # Start server for Functions integration tests.
+    ./Functions/Backend/start.sh synchronous <<!
+myproject
```

Comment:
Wow, I didn't know this was possible! Can we do this inside the `start.sh` script instead when it invokes `"${FUNCTIONS_BIN}"` the first time? It'd be great if no one using the script has to worry about this.

Also, I'm not sure if it works in the macOS version, but apparently in some versions of bash, if you change `<<!` to `<<-!`, then you can indent the next line with tabs, and they will be stripped by the shell, which would make this prettier. Worth a try.